src/languageProvider/locales/en_US.json

```diff
@@ -271,6 +271,8 @@
   "wallet.encryptLabel": "Please enter passphrase to encrypt your wallet",
   "wallet.encryptTitle": "Encrypt Your Wallet",
   "wallet.encrypted": "Your wallet is already encrypted",
+  "wallet.restoreWallet": "Restore Your Wallet",
+  "wallet.restoreWalletDescription": "Please open Qtum Wallet to restore.",
```

Comment:
Can you change to: `Please select Launch Qtum Wallet in the menu and restore your wallet in QT Wallet.`